### PR TITLE
Add Biomed Central harvester

### PR DESCRIPTION
**Pull request by erinspace**

### CODE REVIEW

File: scrapi/harvesters/biomed.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            page += 1
+            records = requests.get(search_url + '&page={}'.format(str(page)), throttle=10)
+            current_records = len(records.json()['entries'])
+
+        return all_records
+
+    def get_contributors(self, record):
+
+        authors = record['authorNames']
+        authors = authors.replace(' <span class="author-names"> ', '').replace('</span> ', '')
```
Comment (chrisseto) [inline scrapi/harvesters/biomed.py]: Just incase the spaces aren't there.
```python
authors = authors.strip().replace('<span class="author-names">', '').replace('</span>', '')
```

File: scrapi/harvesters/biomed.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        authors = authors.split(',')
+
+        if ' and ' in authors[-1]:
+            split_name = authors[-1].split(' and ')
+            authors.pop(-1)
```
Comment (chrisseto) [inline scrapi/harvesters/biomed.py]: Pop returns the popped element so it be a bit better to do
```python
split_name = authors.pop(-1)...
```

File: scrapi/harvesters/biomed.py [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                'ORCID': '',
+            }
+            contributor_list.append(contributor)
+
+        return contributor_list
+
+    def get_ids(self, record):
+
+        return {
+            'serviceID': unicode(record['arxId']),
```
Comment (chrisseto) [inline scrapi/harvesters/biomed.py]: prefer `.encode('utf-8')`

File: scrapi/harvesters/biomed.py [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+            'url': record['articleFullUrl'],
```
Comment (chrisseto) [inline scrapi/harvesters/biomed.py]: Are these already unicode?

File: scrapi/harvesters/biomed.py [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+            'isFree': record['isFree'],
+            'isHighlyAccessed': record['isHighlyAccessed'],
+            'status': record['status'],
+            'abstractPath': record['abstractPath'],
+            'journal Id': record['journal Id'],
+            'published Date': record['published Date']
+        }
+
+    def normalize(self, raw_doc):
+        doc = raw_doc.get('doc')
```
Comment (chrisseto) [inline scrapi/harvesters/biomed.py]: `raw_doc['doc']`

File: scrapi/harvesters/biomed.py [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+        record_list = []
+        for record in records:
+            doc_id = record['arxId']
+
+            record_list.append(
+                RawDocument(
+                    {
+                        'doc': json.dumps(record),
+                        'source': self.short_name,
+                        'docID': unicode(doc_id),
```
Comment (chrisseto) [inline scrapi/harvesters/biomed.py]: prefer `.encode`

File: scrapi/harvesters/biomed.py [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+from nameparser import HumanName
+
+from scrapi import requests
+from scrapi.base import BaseHarvester
+from scrapi.linter.document import RawDocument, NormalizedDocument
+
+logger = logging.getLogger(__name__)
+
+
+class biomedHarvester(BaseHarvester):
```
Comment (chrisseto) [inline scrapi/harvesters/biomed.py]: Always start classes with an uppercase letter even if the word itself would not be capitalized.

File: scrapi/harvesters/biomed.py [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+    short_name = 'biomed'
+    long_name = 'biomed'
```
Comment (chrisseto) [inline scrapi/harvesters/biomed.py]: Looks like it should be BioMed

File: scrapi/harvesters/biomed.py (hunk 4 above)
Comment (erinspace) [inline scrapi/harvesters/biomed.py]: hah, yes!

**Pull request merged**